### PR TITLE
Add ChillCenter.

### PR DESCRIPTION
**Pull request by polo-1245-oficial**
## Type of change

- [x] Add new directory for a server

- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/chillcenter/manifest.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "server_name": "chillcenter",
+  "nice_name": "ChillCenter",
+  "direct_ip": "play.chillcenter.cc",
+  "server_wildcards": [
+    "play.chillcenter.cc"
+  ],
+  "social": {
+    "web": "https://chillcenter.cc",
```
Comment (JakobBruening) [inline minecraft_servers/chillcenter/manifest.json:9]: Currently in maintenance. Update or remove.

**Review by JakobBruening — CHANGES_REQUESTED** [review]
Please compress your background image -> http://compresspng.com

BR, Jakob

File: minecraft_servers/chillcenter/manifest.json (hunk 1 above)
Comment (polo-1245-oficial) [reply minecraft_servers/chillcenter/manifest.json]: Removed.

**Comment by polo-1245-oficial** [comment]
Images test is not working because I compress the image

**Comment by JakobBruening** [comment]
No, the image test is not working because the background size is wrong.
Additionally, the image is even larger than before, which is not the point of compression. Please check what you did wrong. 

BR, Jakob

**Comment by polo-1245-oficial** [comment]
Ready, before I got confused when compressing the images but now they are compressed and ready

**Review by JakobBruening — APPROVED** [review]
Looks good now, thank you! :)

**Pull request merged**